package.json

```diff
@@ -37,11 +37,11 @@
   },
   "homepage": "https://github.com/actyx-contrib/pond-rx-adapter#readme",
   "peerDependencies": {
-    "@actyx/pond": "^2.5",
-    "rxjs": "> 0"
+    "@actyx/pond": "^3",
+    "rxjs": "> 5"
```

Comment:
any reason why we do not support 5 anymore? I know one project, who is using 5.5.12

Reply:
We need `> 5` in `registry`, wouldn't it be good to have the same dependencies?

Reply:
not really, i think you could use rx-pond even if you are not compatible to registry